spec/NumberSpec.php

```diff
@@ -61,6 +63,33 @@ function it_tests_if_the_number_is_integer($number, $decimal)
         $this->isInteger($number)->shouldReturn(!$decimal);
     }
 
+    /**
+     * @dataProvider movePointExamples
+     */
+    function it_moves_point($number, $places, $expected, $integerPart, $fractionalPart)
+    {
+        $this->beConstructedWith($number);
+
+        $this->movePoint($places);
```

Comment:
At the beggining I couldn't find a good name for it, so I just borrowed it from movePointLeft method of the BigDecimal class (Java). If it's also confusing for others then of course I think we can change it to `moveDecimalPlaces` or something more clear.